tool/tctl/common/resource_command.go

```diff
@@ -1151,23 +1172,23 @@ func (rc *ResourceCommand) Delete(ctx context.Context, client auth.ClientI) (err
 		}
 		fmt.Printf("%s '%s/%s' has been deleted\n", types.KindCertAuthority, rc.ref.SubKind, rc.ref.Name)
 	case types.KindKubeServer:
-		kubeServers, err := client.GetKubernetesServers(ctx)
+		servers, err := client.GetKubernetesServers(ctx)
 		if err != nil {
 			return trace.Wrap(err)
 		}
-		deleted := false
-		for _, server := range kubeServers {
-			if server.GetName() == rc.ref.Name {
-				if err := client.DeleteKubernetesServer(ctx, server.GetHostID(), server.GetName()); err != nil {
-					return trace.Wrap(err)
-				}
-				deleted = true
-			}
+		resDesc := "kubernetes server"
```

Comment:
That would return `kube_cluster` instead of `kubernetes cluster`, that's why I did this instead. Following prior art here as well: https://github.com/gravitational/teleport/blob/05ae6df632f2126e2bf7f8a36dc6dd8482262dbf/tool/tctl/common/resource_command.go#L2061